src/com/owncloud/android/datamodel/UploadsStorageManager.java

```diff
@@ -515,14 +516,16 @@ public void updateDatabaseUploadResult(RemoteOperationResult uploadResult, Uploa
                         upload.getOCUploadId(),
                         UploadStatus.UPLOAD_SUCCEEDED,
                         UploadResult.UPLOADED,
-                        upload.getRemotePath()
+                        upload.getRemotePath(),
+                        upload.getStoragePath()
```

Comment:
Will not be so easy; this leads to a crash (later) if behaviour for instant uploads is set to "forget"